### PR TITLE
RR-126 initial timeline views with feature toggle

### PR DESCRIPTION
**Pull request by chrimesdev**
## Description 

Initial Timeline views and tab on the Overview screen with a feature toggle to disable the feature until its fully developed 

screencapture-localhost-3000-plan-A5502DZ-view-timeline-2023-11-14-16_40_36

### CODE REVIEW

File: server/config.ts
Comment (nathanrussell-moj-digital) [inline server/config.ts:136]: Nice 👍 
But I think you _might_ need to add `TIMELINE_PAGE_ENABLED` to the main helm `values.yaml` file (because of the `requiredInProduction` flag, which actually means "required in a deployed environment")
I suspect that if you dont add it to the helm stuff it wont deploy and startup in `dev`

(TBH we should add it anyway for completeness 👍 )

File: server/views/pages/overview/partials/navigation.njk
Comment (nathanrussell-moj-digital) [inline server/views/pages/overview/partials/navigation.njk:41]: 👍

**Review by nathanrussell-moj-digital — DISMISSED** [review]
LGTM 👍

File: server/views/pages/overview/partials/timelineTab/timelineTabContents.njk
Comment (chrimesdev) [reply server/views/pages/overview/partials/timelineTab/timelineTabContents.njk:4]: There's a Slack thread here which explains why I haven't used it just yet, I think we can reconsider it down the line, what do you think? https://mojdt.slack.com/archives/CH5RUSB27/p1698050331609839

Comment (nathanrussell-moj-digital) [reply server/views/pages/overview/partials/timelineTab/timelineTabContents.njk:4]: Yeah, fair shout, that does not sound like it's production ready, so perhaps your approach is correct here and now 👍 

Interesting side point about `moment` being EoL .... we might need to consider that as we make quite a lot of use of `moment` .... 🤔

File: server/config.ts
Comment (chrimesdev) [reply server/config.ts:136]: Added this

Comment (chrimesdev) [reply server/config.ts:136]: May just need re-approval as I've dismissed your review with this change @nathanrussell-moj-digital

**Review by nathanrussell-moj-digital — DISMISSED** [review]
LGTM 👍

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍 
(might be the 3rd time I've approved this 🤣 )

**Comment by chrimesdev** [comment]
> LGTM 👍 (might be the 3rd time I've approved this 🤣 )

That would be my fault 😄

**Pull request merged**
